keyboards/dyz/dyz_tkl/readme.md

```diff
@@ -0,0 +1,19 @@
+# dyz_tkl
+
+![pcb](https://i.imgur.com/5OYWF0Mh.jpg)
+
+TKL pcb that fits most of the boards, support 2 sides of USB and multiple layouts for F row.
+
+-   Keyboard Maintainer: [dayatz](https://github.com/dayatz)
+-   Hardware Supported: ATMega32u4
+-   Hardware Availability: Local Indonesia GB and will available publicly in the future.
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```